packages/jaeger-ui/src/utils/tracking/conv-sentry-to-ga.tsx

```diff
@@ -247,20 +236,14 @@ function compressCssSelector(selector: string) {
 // The chronological ordering of the breadcrumbs is older events precede newer
 // events. This ordering was kept because it's easier to see which page events
 // occurred on.
-interface ICrumb {
```

Comment:
it seems like we used an abstraction over Raven type (even though abstraction itself is a bit coupled, but an abstraction nonetheless, i.e. could be implemented by something other than Raven). What's the reason to remove it instead of just adapting? Adapting seems to only require optionality on the fields, which would minimize the changes throughout this file.

Reply:
This does make sense. But why I thought of using the `Breadcrumb` interface from Sentry was because this function `convSentryToGa` is called from `trackSentryError` in L192, `packages/jaeger-ui/src/utils/tracking/ga.tsx`. 

So there, the transport options come from the `Event` interface in Sentry, which has a breadcrumb field which uses the `Breadcrumb` interface. I just thought of keeping it uniform everywhere and since that field already uses the same interface, used it here as well.